tooling/nargo_cli/src/cli/fuzz_cmd.rs

```diff
@@ -0,0 +1,489 @@
+                writeln!(writer, "issue with a foreign call: ").expect("Failed to write to stderr");
+                writer
+                    .set_color(ColorSpec::new().set_fg(Some(Color::Red)))
+                    .expect("Failed to set color");
+
+                writeln!(writer, "{message}").expect("Failed to write to stderr");
+                writer.reset().expect("Failed to reset writer");
+            }
+            FuzzingRunStatus::ExecutionFailure { message, counterexample, error_diagnostic } => {
+                write!(writer, "execution ").expect("Failed to write to stderr");
+                writer
+                    .set_color(ColorSpec::new().set_fg(Some(Color::Red)))
+                    .expect("Failed to set color");
+                write!(writer, "failed").expect("Failed to write to stderr");
+                writer.reset().expect("Failed to reset writer");
+                writeln!(writer, " with message:").expect("Failed to write to stderr");
+                writer
+                    .set_color(ColorSpec::new().set_fg(Some(Color::Yellow)))
+                    .expect("Failed to set color");
+
+                writeln!(writer, "{message}").expect("Failed to write to stderr");
+                writer.reset().expect("Failed to reset writer");
+                if let Some((input_map, abi)) = counterexample {
+                    writeln!(writer, "Failing input: ",).expect("Failed to write to stderr");
+                    writer
+                        .set_color(ColorSpec::new().set_fg(Some(Color::Red)))
+                        .expect("Failed to set color");
+                    writeln!(
+                        writer,
+                        "{}",
+                        serialize_to_json(input_map, abi)
+                            .expect("Input map should be correctly serialized with this Abi")
+                    )
+                    .expect("Failed to write to stderr");
+                    writer.reset().expect("Failed to reset writer");
+                    let file_name = "Prover-failing-".to_owned()
+                        + &package.name.to_string()
+                        + "-"
+                        + fuzzing_harness_name;
+                    write_inputs_to_file(
+                        fuzzing_failure_path.clone(),
+                        &file_name,
+                        Format::Toml,
+                        abi,
+                        input_map,
+                    )
+                    .expect("Couldn't write toml file");
+                    writeln!(writer, "saved input to:").expect("Failed to write to stderr");
+                    writer
+                        .set_color(ColorSpec::new().set_fg(Some(Color::Yellow)))
+                        .expect("Failed to set color");
+                    let mut full_path_of_example = fuzzing_failure_path.clone().join(file_name);
+                    full_path_of_example.set_extension(PathBuf::from("toml"));
```

Comment:
After this, we could try to make `full_path_of_example` relative to the current directory so the output is a bit shorter. This is done for example here: https://github.com/noir-lang/noir/blob/7a9c8c13f626575f23d25f9cb8689e44dc1c6116/compiler/fm/src/file_map.rs#L91-L96